README.md

```diff
@@ -27,86 +39,57 @@ two members. Thus we need to enable _sympathetic replication_, such that peers
 who don't strictly need the group content are incentivized to replicate it
 anyway.
 
-This document specifies how group content is organized in a metafeed tree, what
-data must be encrypted and to whom, and how peers replicate group-related
-portions of the tree.
+Additionally, our metafeed structure should not allow other peers to learn
+which groups a peer is in, unless they are also members of the group.
 
-## Terminology
+Use of metafeeds for groups is described formally in Section 3 and illustrated
+in the examples in Section 4.
+
+## 2. Terminology
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
 interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
-## Principles
-
-1. **Group membership should be opaque**
-   - You shouldn't be able to guess who is in a group using public info, such
-   as:
-     - Shard feeds
-     - `groupId` (as ssb-uri)
-2. **Peers should replicate sympathetically**
-   - If a friend has a subfeed dedicated to a group, but I don't belong to that
-   group, it is RECOMMENDED that I replicate that subfeed
-   - May be randomized or subject to sympathy-related parameters
+## 3. Functional Specification
 
-## Specification
-
-This work builds on the [ssb-meta-feeds-spec] (v1).
+Implementations supporting this specification MUST follow the
+[ssb-meta-feeds-spec] version 1.0.
 
 We define two types of feeds that each peer will have:
 
 1. A `group/additions` feed
 2. A "group feed" for each group
 
-```mermaid
```

Comment:
The diagram was moved to the Examples section.

Reply:
I'm not in favour of moving the diagram. For me when reading a spec it's all about having a quick frame to hang the details on. If you info-dump text details on me it's very hard to parse and make sense.

Alt solution - link down in document "see example meta-feed structure

Reply:
Well, we got feedback that the spec was too informal and example driven, so ... we have to do something to make it less informal and example driven. Looking at RFCs, I don't see them writing with examples first, I see that examples are often Appendix or something like that.

I agree the diagram is nice :smiling_face_with_tear: But this is a spec, not a tutorial.